source/administration/config-settings.rst

```diff
@@ -153,6 +153,10 @@ This button resets the configuration settings by reloading the settings from the
 
 The workflow for failover without downing the server is to change the database line in the config.json file, click **Reload Configuration from Disk** then click **Recycle Database Connections** in the Advanced > Database section.
 
+Purge All Caches
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This button purges all the in-memory caches for things like sessions, accounts and channels. Deployments using High Availability will attempt to purge all the servers in the cluster. Purging the caches may adversly impact performance.
```

Comment:
`for things like sessions, accounts and channels`  --> should we consider using another term for `things`?

`An option to purge all in-memory caches, including for sessions, accounts and channels.`